index.html

```diff
@@ -276,6 +276,45 @@ <h3 class="im-qaisar">Glad to see you!</h3>
           </ul>
         </div>
     </section>
+    <section>
+      <div class="contact-me-form-container">
+        <h1 class="contact-me-form-heading">Contact me</h1>
+        <p class="contact-me-form-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I'd love to help
+          with it
+        </p>
+        <form  class="contact-me-form">
```

Comment:
- [ ] Kindly as suggested in the project requirements, We are suggested to collect the form's data using formspree.io, you may kindly Go to the Formspree website and create an account. 🙏 

For example:
```
<form action="https://formspree.io/[your-formspree-url]" method="post">
  <input type="text" name="name" placeholder="Your Name">
  <input type="email" name="email" placeholder="Your Email">
  <input type="submit" value="Submit">
</form>
```